src/openforms/locations/api/serializers.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.translation import ugettext_lazy as _
```

Comment:
the `u*` variants are deprecated

```suggestion
from django.utils.translation import gettext_lazy as _
```